dex.yaml

```diff
@@ -43,6 +45,7 @@ pipeline:
       mkdir -p ${{targets.destdir}}/usr/bin/
       mkdir -p ${{targets.destdir}}/srv/dex/
       mkdir -p ${{targets.destdir}}/etc/dex
+      mkdir -p ${{targets.destdir}}/var/dex
```

Comment:
I'm not 100% sure what the trade-off's are for creating empty folders in the apk verses downstream when folks create an image.  I.e. if folks use apko like chainguard images, they tend to create folders + the necessary permissions in the apko.yaml.

Reply:
It is better to create required layout with the package than in the image config.  Consider non-apko uses of the packages, for example :)